tests/e2e/auth/cred.go

```diff
@@ -99,6 +99,21 @@ func createPodPullingFromACR(registryName, image, tag string) (result *v1.Pod) {
 					Image: registryName + ".azurecr.io/" + image + ":" + tag,
 				},
 			},
+			NodeSelector: map[string]string{
+				"kubernetes.io/os": "linux",
+			},
+			Tolerations: []v1.Toleration{
+				{
+					Key:      "node-role.kubernetes.io/control-plane",
```

Comment:
There's defined var for this.
https://github.com/kubernetes-sigs/cloud-provider-azure/blob/21a6572fca7bc25aa6a3f0672a28fc31709030d1/tests/e2e/utils/node_utils.go#L36

Reply:
OK! I will fix that.

Reply:
I have fixed that. PTAL.